lightning/src/ln/channelmanager.rs

```diff
@@ -281,6 +281,7 @@ pub(super) struct PendingAddHTLCInfo {
 	// Note that this may be an outbound SCID alias for the associated channel.
 	prev_short_channel_id: u64,
 	prev_htlc_id: u64,
+	prev_channel_id: Option<ChannelId>,
```

Comment:
I wonder whether this could be non-optional here. Could it be that channel_id is nonexistent here?